client/app/telecom/telephony/alias/configuration/members/telecom-telephony-alias-configuration-members.html

```diff
@@ -7,12 +7,12 @@
         <h1 data-translate="telephony_alias_configuration_members_title"></h1>
```

Comment:
This whole file will be rewrite in a later feature, I've just changed the controller name.